build.ps1

```diff
@@ -746,4 +732,46 @@ if ($packageType -eq 'msixbundle') {
     Write-Host -ForegroundColor Green "`ntar.gz file is created at $tarFile"
 }
 
-$env:RUST_BACKTRACE=1
+function Submit-DSCWinGetAssets {
+    param(
+        [string]$GitToken,
+        [switch] $IsPreRelease
+    )
+
+    $project = 'PowerShell/DSC'
+    # TODO: Package identifier might change in the future
+    $packageId = 'Microsoft.DSC'
+    $restParameters = @{
+        SslProtocol = 'Tls13'
+        Headers     = @{'X-GitHub-Api-Version' = '2022-11-28' }
+    }
+
+    $assets = ((Invoke-RestMethod -uri "https://api.github.com/repos/$Project/releases" -Headers $restParameters) | 
+        Where-Object -Property prerelease -EQ $IsPreRelease.IsPresent |
+         Select-Object -First 1 -Property assets).assets # ExpandProperty did not work
+    # Grab the download URLs for supported WinGet
+    $downloadUrls = $assets.Where({ $_.content_type -in @('application/zip', 'application/octet-stream') }).browser_download_url
```

Comment:
This will break if a new zip based asset is added in the future. I suggest regex matching specific installers from the assets. One can refer to the sample workflows I shared above on how